client/src/components/AuditBoard/ballotInterpretation.ts

```diff
@@ -0,0 +1,65 @@
+import { IBallotInterpretation, Interpretation } from '../../types'
+
+export const INVALID_WRITE_IN = 'INVALID_WRITE_IN'
+
+/**
+ * A modified representation of a ballot interpretation, optimized for the audit board form
+ */
+export interface IBallotInterpretationFormRepresentation {
```

Comment:
Nit: I think `IBallotInterpretationFormState` would be a slightly more conventional name for this